biz.aQute.bndlib.tests/src/test/BuilderTest.java

```diff
@@ -41,6 +41,26 @@
 @SuppressWarnings("resource")
 public class BuilderTest extends BndTestCase {
 
+    /*
+     * #1172 a warning being generated when a package is split between two
+     * jar files that are both in the bundle classpath, and so part of the
+     * same bundle.
+     */
+    public void test1172SplitPackageWarningsInBundleClasspath() throws Exception {
+		Builder A = new Builder();
+        try {
+            A.addClasspath(new Jar(new File("jar/sp-tc-jar1.jar")));
+            A.addClasspath(new Jar(new File("jar/sp-tc-jar1.jar")));
```

Comment:
is this really meant? Probably it should be A.addClasspath(new Jar(new File("jar/sp-tc-jar2.jar")));